docs/guides/helmreleases.md

```diff
@@ -62,12 +62,12 @@ Charts from Git repositories can be released by declaring a
 repository on an interval and expose it as an artifact.
 
 The source-controller can build and expose Helm charts as artifacts
-from the contents of the `GitRepository` artifact (more about this
-later on in the guide).
+from the contents of the `GitRepository` artifact [(more about this
+later on in the guide)](#define-a-helm-release).
 
-**There is one caveat you should be aware of:** to make the
-source-controller produce a new chart artifact, the `version` in the
-`Chart.yaml` of the chart must be bumped.
+!!! warning "Arbitrary changes in Git do not produce new charts."
```

Comment:
```suggestion
!!! warning "Arbitrary changes in Git do not produce new charts"
```

We do not end our warning block titles with a dot, as they are seen as headings.